src/EnergyPlus/MixedAir.cc

```diff
@@ -1670,14 +1670,17 @@ namespace MixedAir {
 				for ( groupNum = 1; groupNum <= NumGroups; ++groupNum ) {
 					VentMechZoneName( groupNum ) = AlphArray( ( groupNum - 1 ) * 3 + 5 );
 
-					for ( OutAirNum = 1; OutAirNum <= NumOAControllers; ++OutAirNum ) {
-						if ( OAController( OutAirNum ).VentilationMechanicalName == VentilationMechanical( VentMechNum ).Name && VentilationMechanical( VentMechNum ).DCVFlag ) {
-							AirLoopControlInfo( OutAirNum ).AirLoopDCVFlag = true;
-						} else {
-							AirLoopControlInfo( OutAirNum ).AirLoopDCVFlag = false;
+					if ( AirLoopControlInfo.size() >= NumOAControllers ){
+						for ( OutAirNum = 1; OutAirNum <= NumOAControllers; ++OutAirNum ) {
+							if ( OAController( OutAirNum ).VentilationMechanicalName == VentilationMechanical( VentMechNum ).Name && VentilationMechanical( VentMechNum ).DCVFlag ) {
+								AirLoopControlInfo( OutAirNum ).AirLoopDCVFlag = true;
```

Comment:
@JasonGlazer Looking into this a little, one can't assume that the order of OA controllers and the order of airloops are the same.  When [SimOAController is called from an airloop](https://github.com/NREL/EnergyPlus/blob/c9755cea9f63a0b515ea12331e3b7f0b91ed2796/src/EnergyPlus/MixedAir.cc#L523) the airloopnum is passed in.  When [SimOAController is called from an ERV](https://github.com/NREL/EnergyPlus/blob/c9755cea9f63a0b515ea12331e3b7f0b91ed2796/src/EnergyPlus/HVACStandAloneERV.cc#L1153) a zero is passed for the airloopnum. Given that the OA controllers are likely processed early for ERVs here, the airloops probably aren't gotten yet.  I'll try to make a pair of defect files that show this code is wrong even if there aren't any ERVs.  Seems if this structure is kept here, there needs to be another loop over AirLoopControlInfo to determing the corresponding airloopnum, checking AirLoopControlInfo.OACtrlNum to see which airloop this OAcontroller is on.

Also noticed several stray using MixedAir::SimOAController statements that should be deleted (in SimAirServingZones and HVACVariableRefrigerantFlow).